### ISSUE TITLE
external script uses python3 cause error

### ISSUE THREAD

**Issue by lenovore** [signal18/replication-manager#373]
failover-post-script = "/data/replication-manager/scripts/aws_vip_helper.py"

![QQ截图20210817160011](https://user-images.githubusercontent.com/29646774/129687670-7148946f-7c7e-46a2-84b7-73823db18ca2.png)

But using a shell script is normal，Doesn't it support calling python?

**Comment by svaroqui** [comment]
Hi lenovore , 

Thanks for using replication-manager 

that's expected as .py files are interpreted and in go all argument need to be split in seperate args to exec call 

Can you try something like a bash wrapper : 
cat /data/replication-manager/scripts/post_failover.sh 
```
#!/bin/bash
python /data/replication-manager/scripts/aws_vip_helper.py "$@"
```

`chmod +x /data/replication-manager/scripts/`

failover-post-script = "/data/replication-manager/scripts/post_failover.sh"

svar

**Comment by svaroqui** [comment]
Re, 

If you don't mind sharing your script i can include both scripts inside a new share/failover_scripts directory so that we start a repo of helper systems scripts  ; i thing i get some for OVH VIP failover as well that i don't expose. 

svar

**Comment by lenovore** [comment]
I tried it and it’s okay to use bash wrapper. I just want to confirm if python can’t be called directly.
The vip failover script I wrote is relatively simple, for reference only
[aws_vip_helper.zip](https://github.com/signal18/replication-manager/files/6998428/aws_vip_helper.zip)

**Comment by svaroqui** [comment]
Please note that 2.2 has also start stop provision unprovision callable external scripts if you'd like using 
- rolling restart feature 
- cluster configurator 
Let me know if you would like some extra scripts as well

**Comment by lenovore** [comment]
I am currently using version 2.0.2. 
The new version adds some new features, but the help documentation does not seem to be updated synchronously. I have tested the 2.1.1 version before, and some of the configurations are not well understood, and it seems that the web UI is more consuming RAM

**Comment by svaroqui** [comment]
This is fixed now and i think that  2.1 is really stable i came to change some default to get no extra work feature
